roles/katello_provisioning/tasks/main.yml

```diff
@@ -102,6 +102,35 @@
     --organization "{{ katello_provisioning_organization }}"
   when: katello_provisioning_sync_repos
 
+# EPEL
+- name: 'find epel repo'
+  shell: >
+    {{ katello_provisioning_hammer }} repository info
+    --name "EPEL 7"
+    --product "CentOS"
+    --organization "{{ katello_provisioning_organization }}"
+  register: katello_epel_repo
+  ignore_errors: True
+
+- name: 'create epel repo'
+  shell: >
+    {{ katello_provisioning_hammer }} repository create
+    --product="CentOS"
+    --content-type="yum"
+    --name "EPEL 7"
+    --url "http://download.fedoraproject.org/pub/epel/7/x86_64"
+    --download-policy {{ katello_provisioning_download_policy }}
+    --organization "{{ katello_provisioning_organization }}"
+  when: katello_epel_repo.stderr.find('not found') != -1
+
+- name: 'sync the epel repo'
+  shell: >
+    {{ katello_provisioning_hammer }} repository synchronize
+    --name "EPEL 7"
```

Comment:
This just ate my hard-drive: do we really need to download 12GB of data by default? I would perhaps leave it as an optional thing to do (something like `katello_provisioning_epel_sync`).

Reply:
how did you test it? default policy is on_demand so unless you do it with immediate it should be fine (that is default for demo hw, perhaps you tested it with that?)

Reply:
You are right, I haven't realized this: I actually panicked, because I was low on disk space, and even the metadata parsing probably caused some spikes. So I guess we are ok now :)